src/SayMore/UI/ConvertMediaDlg.cs

```diff
@@ -157,7 +148,6 @@ private void SetupForAutoRun()
 		/// ------------------------------------------------------------------------------------
 		private void HandleStringsLocalized()
 		{
-			_labelDownloadNeeded.Tag = _labelDownloadNeeded.Text;
 			UpdateDisplay();
```

Comment:
This method is probably no longer needed. We'd only need to call UpdateDisplay when the localized strings change if we're doing something related to the strings (e.g., setting a variable or property to the new value).

Reply:
Done.